models/xero__profit_and_loss_report.sql

```diff
@@ -1,3 +1,23 @@
+{%- set using_tracking_categories = (
+    var('xero__using_journal_line_has_tracking_category', True)
```

Comment:
Same note about the variable inconsistency between files and source and transform.

Reply:
Updated.